content/applications/finance/accounting/payments/internal_transfers.rst

```diff
@@ -77,38 +72,30 @@ Automated booking - Bank journal (BANK B)
      - **$1,000**
 
 .. note::
-   There are one outstanding payment and one outstanding receipt pending in your two bank account
+   There is one outstanding payment and one outstanding receipt pending in your two bank account
```

Comment:
If you do decide to have the tables in an example box, I'd remove the note and include the text in the example box after the tables.